pkg/sfu/publisher.go

```diff
@@ -63,43 +69,15 @@ func NewPublisher(session Session, id string, relay bool, cfg *WebRTCTransportCo
 		r, pub := p.router.AddReceiver(receiver, track)
 		if pub {
 			p.session.Publish(p.router, r)
-		}
-
-		if relay && cfg.Relay != nil && pub {
-			codec := track.Codec()
-			downTrack, err := NewDownTrack(webrtc.RTPCodecCapability{
-				MimeType:     codec.MimeType,
-				ClockRate:    codec.ClockRate,
-				Channels:     codec.Channels,
-				SDPFmtpLine:  codec.SDPFmtpLine,
-				RTCPFeedback: []webrtc.RTCPFeedback{{"goog-remb", ""}, {"nack", ""}, {"nack", "pli"}},
-			}, r, cfg.BufferFactory, id, cfg.Router.MaxPacketTrack)
-			if err != nil {
-				Logger.V(1).Error(err, "Create Relay downtrack err", "peer_id", id)
-				return
-			}
-			rr, sdr, err := cfg.Relay.Send(session.ID(), id, receiver, track, downTrack)
-			if err != nil {
-				Logger.V(1).Error(err, "Relay err", "peer_id", id)
-				return
-			}
-
-			if p.relayPeer == nil {
-				relayReports.Do(func() {
-					p.relayPeer = rr
-					go p.relayReports()
-				})
-			}
-
-			downTrack.OnCloseHandler(func() {
-				if err := sdr.Stop(); err != nil {
-					Logger.V(1).Error(err, "Relay sender close err", "peer_id", id)
+			p.mu.Lock()
+			p.tracks = append(p.tracks, publisherTracks{track, r})
+			for _, rp := range p.relayPeer {
+				if err = p.createRelayTrack(track, r, rp); err != nil {
+					Logger.V(1).Error(err, "Error creating relay track", "peer_id", p.id)
```

Comment:
```suggestion
					Logger.V(1).Error(err, "Creating relay track.", "peer_id", p.id)
```

Redundant to include "error" in errors/error messages